portal/src/main/webapp/js/src/DataProxyFactory.js

```diff
@@ -41,7 +41,7 @@ var DataProxyFactory = (function()
 	 *
 	 * @return singleton instance of MutationDataProxy class
 	 */
-	function getDefaultMutationDataProxy()
+	function getDefaultMutationDataProxy(geneticProfiles,caseSetId,caseIdsKey,caseList)
```

Comment:
Should we also support the case where no parameter is passed to this function?

For an example, see the comments below for each individual parameter.